### PR TITLE
24 pixel icons

### PR DESCRIPTION
**Pull request by Roohn**
# Description
For icons that have the size of 24 pixels or increments of it, there is now a custom SVG made.
Reason for this is that 32 pixel icons causes our icons to be blurry on non-retina screens, because we scale most of the icons to 24 pixels anyway.

image

This closes issue https://github.com/TicketSwap/comets/issues/373

### CODE REVIEW

File: src/Icon.js [hunk 1]
```diff
@@ -29,22 +29,48 @@ const InlineSvg = styled.svg`
...
+      <InlineSvg
+        aria-label={a11yTitle}
+        fillRule="evenodd"
+        clipRule="evenodd"
+        strokeLinejoin="round"
+        strokeMiterlimit="1.414"
+        xmlns="http://www.w3.org/2000/svg"
+        role="presentation"
+        focusable="false"
+        viewBox="0 0 32 32"
```
Comment (RobVermeer) [inline src/Icon.js:68]:
```suggestion
        viewBox={isSmallIcon ? '0 0 24 24' : '0 0 32 32'}
```

So we don't need to duplicate the rest?

File: src/Icon.js [hunk 2]
```diff
@@ -29,22 +29,48 @@ const InlineSvg = styled.svg`
...
+          focusable="false"
+          viewBox="0 0 24 24"
+          preserveAspectRatio="xMidYMid meet"
+          fit
+        >
+          {children({ isSmallIcon })}
+        </InlineSvg>
+      </SvgWrapper>
+    )
+  }
```
Comment (RobVermeer) [inline src/Icon.js:55]: See other comment, don't think this is needed.

**Comment by RobVermeer** [comment]
Do you know what the build size change is? I have a feeling that this will substantially impact the build size...

**Comment by Roohn** [comment]
> Do you know what the build size change is? I have a feeling that this will substantially impact the build size...

I will have a look, was thinking the same 😬

**Comment by Roohn** [comment]
Closing this because it has a major impact on bundle size

**Pull request closed without merging**